### ISSUE TITLE
http://llvm.org/git/xxxx.git can't successful visit

### ISSUE THREAD

**Issue by MageWeiG** [Dor1s/libfuzzer-workshop#15]
when I install LLVM for Libfuzzer, someting is wrong.

when i run "checkout_build_install_llvm.sh" , and return some wrongs, it say that "Git.llvm.org Excessive redirection times."

and i check checkout_build_install_llvm.sh, and found the code

"
cd $WORK_DIR/src && git clone --depth 1 http://llvm.org/git/llvm.git
cd $WORK_DIR/src/llvm/tools && git clone --depth 1 http://llvm.org/git/clang.git
cd $WORK_DIR/src/llvm/projects && git clone --depth 1 http://llvm.org/git/compiler-rt.git
cd $WORK_DIR/src/llvm/projects && git clone --depth 1 http://llvm.org/git/libcxx.git
cd $WORK_DIR/src/llvm/projects && git clone --depth 1 http://llvm.org/git/libcxxabi.git
"
then i visit the "http://llvm.org/git/xxx.git" in my browser，and also can't visit successfully.

May I ask whether this is my operation error or this website has updated these urls?

**Comment by Dor1s** [comment]
@MageWeiG you're right, LLVM migrated to GitHub a couple of years ago: https://github.com/llvm/llvm-project

A newer version of the script for checking out and building LLVM can be found in the OSS-Fuzz repo: https://github.com/google/oss-fuzz/blob/master/infra/base-images/base-clang/checkout_build_install_llvm.sh

However, I cannot guarantee that it will work as-is. Please review it before running.
